packages/attestation-service/src/db.ts

```diff
@@ -22,11 +23,31 @@ export let kit: ContractKit
 export async function initializeKit() {
   if (kit === undefined) {
     kit = newKit(fetchEnv('CELO_PROVIDER'))
-    const blockNumber = await kit.web3.eth.getBlockNumber()
-    if (blockNumber === 0) {
-      throw new Error(
-        'Could not fetch latest block from web3 provider ' + fetchEnv('CELO_PROVIDER')
+    // Copied from @celo/cli/src/utils/helpers
```

Comment:
would be better to have a function for this

Reply:
I thought about it, but we only have `utils` and I did not want to add `web3` as a dependency